utils/createCompanyList.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components'
+import { ColumnDef } from '@tanstack/react-table'
+import { Company } from './types'
+
+const ScopeColumn = styled.span<{ isMissing: boolean }>`
+  color: ${({ isMissing, theme }) => (isMissing ? 'gray' : theme.darkYellow)};
+  font-style: ${({ isMissing }) => (isMissing ? 'italic' : 'normal')};
+`
+
+export const companyColumns = (): ColumnDef<Company>[] => [
+  {
+    header: 'Företag',
+    cell: (row) => {
+      const company = row.cell.row.original
+      return company.Url ? (
+        <a
+          href={company.Url}
+          target="_blank"
+          rel="noopener noreferrer"
+          style={{ cursor: 'pointer' }}
+        >
+          {company.Name}
+        </a>
+      ) : (
+        <span>{company.Name}</span>
+      )
+    },
+    accessorKey: 'Name',
+  },
+  {
+    header: 'Egna utsläpp (tCO₂e)',
+    cell: (row) => {
+      const scope1n2Emissions = row.cell.row.original.Emissions.Scope1n2
+      const scope1n2String = scope1n2Emissions ? scope1n2Emissions.toString() : 'Ej rapporterat'
```

Comment:
Would be nice to use Intl.NumberFormatter` here to get locale-aware number formatting (incl. Swedish for those who prefer it).

Since number formatting is  needed in other places in the website too, it would be nice to break it out into a utility function like `formatNumber()`